heron/stmgr/src/cpp/server/stmgr-main.cpp

```diff
@@ -70,8 +70,15 @@ int main(int argc, char* argv[]) {
     LOG(FATAL) << "Corrupt topology defn file" << std::endl;
   }
 
+  sp_int64 high_watermark = heron::config::HeronInternalsConfigReader::Instance()
+                              ->GetHeronStreammgrNetworkBackpressureHighwatermarkMb() *
+                                1024 * 1024;
```

Comment:
Yeah, we can do it in a separate PR, because there are multiple places using the same.